src/helpers/index.ts

```diff
@@ -0,0 +1,10 @@
+export {default as FormData} from './FormData'
+export {default as isAxiosResponse} from './isAxiosResponse'
+export {default as isBrowser} from './isBrowser'
+export {default as isContentTree} from './isContentTree'
+export {default as isSubmissionTree} from './isSubmissionTree'
+export {default as path} from './path'
+export {default as requiredArg} from './requiredArg'
+export {default as URL} from './URL'
+export {default as URLSearchParams} from './URLSearchParams'
+export {default as WebSocket} from './WebSocket'
```

Comment:
maybe named export would suit better here?

Reply:
@DedaDev Idk.. Yeah, this file isn't necessary.